lib/build.js

```diff
@@ -41,7 +41,20 @@ function getBabelTarget(envConfig) {
   return unknown ? '8.15.0' : current.replace(/^nodejs/, '');
 }
 
+function getRepositoryRoot(functionsDir) {
+  const gitDirectory = findUp.sync('.git', {
+    cwd: functionsDir,
+    type: 'directory',
+  });
+  if (gitDirectory === undefined) {
+    return functionsDir;
```

Comment:
Would it not make more sense to fall back to `process.cwd` here?  
Having a babel config file in the root directory and a functions folder in a subdirectory would otherwise fail to find the babel config file in a non-git project

Reply:
Yes it would, good catch!